src/simd/simd.h

```diff
@@ -31,8 +31,8 @@
 #ifndef __QUAD_SIMD_SIMD_H__
 #define __QUAD_SIMD_SIMD_H__
 
-#include "simd/allocator.h"
-#include "simd/definitions.h"
+#include "allocator.h"
```

Comment:
That's weird, then something is probably wrong in the install, not in the include directive.

Reply:
When compiling quadiron-tools:
```
/usr/local/include/quadiron/simd/simd.h:34:10: fatal error: simd/allocator.h: No such file or directory
 #include "simd/allocator.h"
          ^~~~~~~~~~~~~~~~~~
```
I put it aside for now, we need to solve it later.

Reply:
actually checked, it is good to do that